raritan/testing/fixture/test.txt

```diff
@@ -0,0 +1 @@
+A tiny fixture for testing IO.
```

Comment:
Is this a work-in-progress?

Reply:
Nope. It's just something to load in our tests.